resources/assets/js/app/components/MainNavigation.js

```diff
@@ -52,7 +54,7 @@ class MainNavigation extends Component {
 
                 <ul className="nav navbar-nav navbar-right">
                   <li className="nav-item">
-                    <Link to="#" className="nav-link">Login</Link>
+                    <Link to={ logInUrl } className="nav-link">{ logInMsg }</Link>
```

Comment:
should be 2 different navigation components: for lgged in user and for not logged in user